.github/workflows/build_package.yml

```diff
@@ -1,83 +1,83 @@
-
-    - name: Install tree CLI tool
-      run: sudo apt-get install -y tree
-
-    - name: Display directory tree before building
-      run: tree
-
-    - name: Set up Conda
-      uses: conda-incubator/setup-miniconda@v2
-      with:
-        environment-file: env.yml
-        activate-environment: mle-dev
-
-    - name: Building the package
-      run: |
-        pip install --upgrade setuptools
-        pip install --upgrade build
-        python -m build
-
-    - name: Display directory tree after building
-      run: tree
-
-    - name: Check contents of dist directory
-      run: ls -la
-
-    - name: Upload build artifacts
-      uses: actions/upload-artifact@v3
-      with:
-        name: build-artifacts
-        path: dist/*
-
-  deploy_package:
-    runs-on: ubuntu-latest
-    defaults:
-      run:
-        shell: bash -l {0}
-    needs: build_package
-    steps:
-    - name: Checkout code
-      uses: actions/checkout@v3
-
-    - name: Set up Conda
-      uses: conda-incubator/setup-miniconda@v2
-      with:
-        environment-file: env.yml
-        activate-environment: mle-dev
-
-    - name: Download build artifacts
-      uses: actions/download-artifact@v2
-      with:
-        name: build-artifacts
-        path: dist  
-
-    - name: List downloaded artifacts
-      run: ls -la dist
-
-    - name: Install the package
-      run: |
-        pip install dist/*.whl
-
-    - name: Run housing_price.py
-      run: |
-        python -m house_package_saachi.housing_price
-
-    - name: Display directory tree after installation
-      run: tree
+name: Build and Deploy as Python module
+
+on:
+  pull_request:
+    branches: 
+      - dummy_branches
+
+jobs:
+
+  build_package:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Install tree CLI tool
+      run: sudo apt-get install -y tree
+
+    - name: Display directory tree before building
+      run: tree
+
+    - name: Set up Conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: env.yml
+        activate-environment: mle-dev
+
+    - name: Building the package
+      run: |
+        pip install --upgrade setuptools
+        pip install --upgrade build
+        python -m build
+
+    - name: Display directory tree after building
+      run: tree
+
+    - name: Check contents of dist directory
+      run: ls -la
+
+    - name: Upload build artifacts
+      uses: actions/upload-artifact@v3
+      with:
+        name: build-artifacts
+        path: dist/*
+
+  deploy_package:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash -l {0}
+    needs: build_package
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+
+    - name: Set up Conda
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        environment-file: env.yml
+        activate-environment: mle-dev

```

Comment:
This part has to be changed to `Miniforge` instead of `Miniconda`.
Please refer to this [Example-5](https://github.com/marketplace/actions/setup-miniconda#example-5-custom-installer).

Reply:
Changed to Miniforge as directed